components/home/FAQSection.vue

```diff
@@ -35,20 +35,13 @@
 </template>
 
 <script setup lang="ts">
+import { CirclePlusIcon, CircleMinusIcon } from 'vue-tabler-icons'
 import { marked } from 'marked'
 import sanitizeHtml from 'sanitize-html'
 
-type FAQ = {
-  question: string
-  answer: string
-}
+const answers = ref<string[]>([])
 
-const props = defineProps({
-  faqs: {
-    type: Array<FAQ>,
-    required: true,
-  },
-})
+const props = defineProps(['faqs'])
```

Comment:
Same. Not sure why prefer the flat syntax as it removed type inference and removed the default value of empty array.